### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by erickt** [pfernie/user_agent#1]
Hello @pfernie! I had some free time, so I wanted to try to push the whole Rust cookie session management story a little down the road. I was wondering if you could put a license on this project. I suggest MIT/Apache 2.0, which is what most of our stuff uses, and would make it easy to have this code be used by hyper/reqwest and etc. But it of course up to you. Thanks!

**Comment by pfernie** [comment]
I've added a license (MIT/Apache 2.0). I also added some notes in the [README.md](README.md) about where this project lies/future direction. Basically, it'd be great if work here can be leveraged elsewhere; this repo has obviously stagnated since being put up. As mentioned, I have some questions about the best way to go forward. I'm happy to contribute what I can, but honestly would not have the time currently to make anything `crates.io`-worthy. The crate as it exists grew relatively organically, and as I was learning rust, so is pretty warty w.r.t. being idiomatic and in terms of API design.
